### DISCUSSION TITLE
Custom Previews?

### DISCUSSION THREAD

**Discussion by YungB329** [stashapp/stash#5240] [Q&A] [answered]
Is it possible right now to create a custom preview?
For example, most artists on iwantclips have already a preview, either on the site of the clip or even the first seconds of the clip. I would like to use those, but if i paste the link of said preview it does not show?
If not this might be a quick and nice feature.

**Comment by stg-annon** [comment, marked as answer]
so yes, previews are stored in `{generated}\screenshots` as a video file the name of the file is the OSHASH or MD5 of the scene it corresponds to stash will look for that file when looking for the preview any file you place there with the correct name will be served as the preview
